src/index.ts

```diff
@@ -461,12 +471,141 @@ async function _startSessionPrograms(
     })
     .map(win => {
       win.instancesStarted += 1;
-      return startProgram(win.executableFile, win.desktopFilePath);
+      return startProgram(
```

Comment:
If I am not mistaken this would lead to`isAllowInputArgs` being false to have no effect?

Reply:
Sorry for taking so long to answer,
The code would start the programs with arguments, in case there were input arguments in the json file, if the `isAllowInputArgs` was set to false.
The effect `isAllowInputArgs` being set to false would be running `_startSessionPrograms` instead of `_startAndWaitPrograms`.
I believe I did that way because the input arguments themselves were not a big change on the code, that could lead to instabilities (like `_startAndWaitPrograms`).
But, now that you said, I agree that makes much more sense to remove them. I will just pass a empty string as an argument then, so the code will behave exactly as before.